src/components/NavigationBar/NavBarIcons.js

```diff
@@ -0,0 +1,137 @@
+import React, {useState, useEffect, useRef} from 'react';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import Icon from './Icon';
+
+const NavBarIcons = ({logout, classes}) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [openIcon, setOpenIcon] = useState(0);
+  const [isWarning, setIsWarning] = useState(false);
+  const wrapperRef = useRef(null);
+
+  const BetaMessage = (
+    <p className={classes.betaMessage}>
+      <span>
+        ** This website is currently in beta. There will be updates to the look
+        and feel and you may experience some bugs as we introduce new
+        features.**
+      </span>
+      <br />
+      <br />
+      <span className={classes.smallerFont}>
+        If you notice an issue or are having a problem with the website,
+        <br /> you can let us know through our{' '}
+        <a
+          className={classes.contactSupportLink}
+          target="_blank"
+          rel="noopener noreferrer"
+          href="https://www.tfaforms.com/4602493"
+        >
+          Contact Support Page
+        </a>
+      </span>
+    </p>
+  );
+
+  const helpLink = (
+    <a
+      className={classes.helpLink}
+      target="_blank"
+      rel="noopener noreferrer"
+      href="https://www.tfaforms.com/4602493"
+    >
+      Help
+    </a>
```

Comment:
Since you use this anchor tag in two places, it might make sense to wrap it as a small reusable component that takes some parameters

Reply:
I forgot that I have that function in another branch already, connect-programs-to-redux branch (the most updated branch so far). Gonna copy it to this branch. Thanks